### PR TITLE
clojure: fix livecheck URL

### PR DESCRIPTION
**Pull request by kakuhen**
#### Description
The content that the livecheck regex matches for is in a new page. This commit allows users to run `port livecheck` again. Since this change does not influence the Clojure installation in any way, I decided to not bump the revision number.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.7 19H1519 x86_64
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by jtrtik** [comment]
thanks...

**Review by mascguy — APPROVED** [review]
LGTM, and maintainer provided a thumbs-up. Merging.

**Pull request merged**
